app/script/view_model/bindings/CommonBindings.js

```diff
@@ -188,11 +186,10 @@ ko.bindingHandlers.resize = (function() {
     },
 
     update(element, valueAccessor) {
-      const params = ko.unwrap(valueAccessor()) || {};
-      triggerValue = params.triggerValue;
-      syncElement = document.querySelector(params.syncElement);
```

Comment:
Another perf improvement here, we now don't query the DOM on every single input (it is done when the binding is initiated)